tsconfig.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
what's this root tsconfig for?

Reply:
Better type resolution while editing just.config.ts throughout the repo